_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -470,6 +471,7 @@ def get(
     ):
         with self.client.get(url=url, name=grouping, allow_redirects=False, catch_response=True) as response:
             self.verify_response(expected_response_status, expected_response_text, response, url)
+            time.sleep(r.randint(USER_WAIT_TIME_MIN_SECONDS, USER_WAIT_TIME_MAX_SECONDS))
```

Comment:
Now these waits are actually per request, we'll probably need to reduce them or increase the number of users in the daily tests to keep the same number of requests as before. We'll tweak it via Concourse once it's merged.